Pulsar basic and token auth (#1179)
emqtt/emq-docs-cn: en_US/rule/bridge_pulsar.md

Configure the resource:
```

> Set the "Pulsar Server" to "pulsar://pulsar-broker1:6650" (multiple servers should be separated by comma), and keep all other configs as default, and click on the "Test" button to make sure the connection can be created successfully, and then click on the "Create" button.

Reviewer: ```suggestion
Set the "Pulsar Server" to "pulsar://pulsar-broker1:6650" (multiple servers should be comma-separated), and keep the default values for all other configurations. Click the "Test" button to make sure the connection can be created successfully, and then click the "Create" button.
```
?

> The `Pulsar Server` should be Pulsar URLs, in format: `pulsar://<hostname>:<port>`, or `pulsar+ssl://<hostname>:<port>` if the TLS is enabled on the Pulsar side.

Reviewer: ```suggestion
`Pulsar Server` should be comma-separated Pulsar URLs, in format: `pulsar://<hostname>:<port>`, or `pulsar+ssl://<hostname>:<port>` if TLS is enabled on the Pulsar side.
```

![image](./assets/rule-engine/pulsar_rule_overview_1.png)

## Use basic/token authentication for pulsar

> After EMQX Enterprise 4.3.10 and 4.4.4 we support `basic` and `token` authentication to the pulsar server.

Reviewer: ```suggestion
Starting at EMQX Enterprise 4.3.10 and 4.4.4, we support `basic` and `token` authentication when connecting to Pulsar.
```
?
